unit-tests/py/rspy/unittest.py

```diff
@@ -0,0 +1,367 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2021 Intel Corporation. All Rights Reserved.
+
+import re, os, subprocess, time, sys
+from abc import ABC, abstractmethod
+
+from rspy import log, file
+
+# this script is in unit-test/py/rspy
+unit_tests_dir = os.path.dirname( os.path.dirname( os.path.dirname( os.path.abspath( __file__ ) ) ) )
+# the full path to the directory that should hold the unit-tests logs. It is updated in run-unit-tests when we know
+# the target directory
+logdir = None
+# if True all the messages are to be printed to stdout and not to a log file. Updated in run-unit-tests if
+# the flag specifying this option is given
+to_stdout = False
```

Comment:
The way it looks is that, by default, `logdir` is `None` and `to_stdout` is False? What's the behavior then?

Reply:
The only issue I see is that if we use `test.get_log()` we get a error. Which makes sense. Other than that don't see anything special since running a test requires a `logpath`.

Reply:
If `logdir` is None, what happens when you try to run a test?

Reply:
So you want `to_stdout` to start as `True` and become `False` only if there is a `logdir`?

Reply:
That would just mean that in run-unit-tests I turn it to false before checking the flags because the default for run-unit-tests is that it's off

Reply:
Why not combine into one parameter?
If `logdir` is None, output to stdout -- otherwise, log to files?
Do we ever need both? Or neither?